src/client/components/media-video.vue

```diff
@@ -35,71 +34,93 @@ export default defineComponent({
 	data() {
 		return {
 			hide: true,
-			faPlayCircle,
 			faExclamationTriangle,
-			faEyeSlash
+			faEyeSlash,
+			player: null,
 		};
 	},
 	computed: {
-		imageStyle(): any {
-			return {
-				'background-image': `url(${this.video.thumbnailUrl})`
-			};
-		}
 	},
 	created() {
 		this.hide = (this.$store.state.nsfw === 'force') ? true : this.video.isSensitive && (this.$store.state.nsfw !== 'ignore');
 	},
+	mounted() {
+		this.$nextTick(() => {
+			this.player = videojs(this.$refs.videoCast, {
+				autoplay: false,
+				controls: true,
+				preload: "metadata",
```

Comment:
I think this may incrases traffic of media server because browser preloading all videos, so I suggest `preload: "none"`